pkg/phases/konfigadm.go

```diff
@@ -102,6 +104,15 @@ func addInitKubeadmConfig(platform *platform.Platform, cfg *konfigadm.Config) er
 	}
 	platform.Tracef("Using kubeadm config: \n%s", string(data))
 	cfg.Files["/etc/kubernetes/kubeadm.conf"] = string(data)
+	if platform.Kubernetes.AuditConfig.PolicyFile != "" {
+		// for vSphere clusters audit policy files are injected into the machine
+		// via the KonfigAdm
```

Comment:
```suggestion
		// clusters audit policy files are injected into the machine via konfigadm
```

So this phase doesn't actually know what it is being used for, e.g. it could be sphere today, and AWS in the future